scripts/start.sh

```diff
@@ -87,16 +87,16 @@ done
 
 if [[ "$FORCE_PROD_MODE" == "True" ]]; then
   constants_env_variable="\"DEV_MODE\": false"
-  sed -i.bak -e s/"\"DEV_MODE\": .*"/"$constants_env_variable"/ assets/constants.js
+  sed -i.bak -e s/"\"DEV_MODE\": .*"/"$constants_env_variable"/ assets/constants.ts
```

Comment:
Hi @brianrodri, there is no file as `constants.ts`. The constants file is maintained as `.js` in assets and so this part was not changed. 

Is there any specific issue happening while running server in prod mode?